From e78cc1db2ddde083089c072e38d4843155ce6aa1 Mon Sep 17 00:00:00 2001
From: Shane Freeder <theboyetronic@gmail.com>
Date: Tue, 17 Jun 2025 15:04:22 +0100
Subject: [PATCH] Temp release support


diff --git a/protocol/src/main/java/net/md_5/bungee/protocol/ProtocolConstants.java b/protocol/src/main/java/net/md_5/bungee/protocol/ProtocolConstants.java
index f8be48b9..5405096c 100644
--- a/protocol/src/main/java/net/md_5/bungee/protocol/ProtocolConstants.java
+++ b/protocol/src/main/java/net/md_5/bungee/protocol/ProtocolConstants.java
@@ -121,7 +121,7 @@ public class ProtocolConstants
                 ProtocolConstants.MINECRAFT_1_21_7
         );
 
-        if ( SNAPSHOT_SUPPORT )
+        if ( true || SNAPSHOT_SUPPORT )
         {
             // supportedVersions.add( "1.21.x" );
             // supportedVersionIds.add( ProtocolConstants.MINECRAFT_1_21_6 );
-- 
2.43.0

